src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -331,15 +331,17 @@ public function getModelData($form_data, $isMultiShopContext = false)
         if (!empty($form_data['features'])) {
             foreach ($form_data['features'] as $dataFeature) {
                 $idFeature = $dataFeature['feature'];
-
-                //custom value is defined
-                if ($dataFeature['custom_value'][$this->defaultLocale]) {
-                    foreach ($this->locales as $locale) {
-                        $form_data['feature_'.$idFeature.'_value'] = null;
-                        $form_data['custom_'.$idFeature.'_'.$locale['id_lang']] = $dataFeature['custom_value'][$locale['id_lang']];
+                
+                if (!empty($idFeature)) {
```

Comment:
Can we have instead

```php
if (empty($idFeature)) {
    continue;
}
```
?

This would prevent the indentation to go too far on the right. :)